storey/table.py

```diff
@@ -12,58 +18,53 @@ class Table:
          can optimize writes. Defaults to True.
         """
 
-    def __init__(self, table_path: str, storage: Driver, partitioned_by_key: bool = True):
+    def __init__(self, table_path: str, storage: Driver, partitioned_by_key: bool = True, use_windows_from_schema: bool = False):
         self._container, self._table_path = _split_path(table_path)
         self._storage = storage
-        self._cache = {}
-        self._aggregation_store = None
+        self._attrs_cache = {}
         self._partitioned_by_key = partitioned_by_key
+        self._aggregates = None
+        self._schema = None
+        self._read_only = False
+        self._use_windows_from_schema = use_windows_from_schema
 
-    def __getitem__(self, key):
-        return self._cache[key]
-
-    def __setitem__(self, key, value):
-        self._cache[key] = value
-
-    def update_key(self, key, data):
-        if key in self._cache:
+    def update_static_attrs(self, key, data):
```

Comment:
I would consider keeping `__getitem__` and `__setitem__` to get/update the "static" fields. and treat the aggregations in a special way by calling `get_features` or something.

This is just more intuitive for users that will use the `Table` object

Reply:
i disagree, i think it should be clear what you're setting now that table is handling both types of attrs
it's not intuitive that the default it static imo